### ISSUE TITLE
Error 500 on settings page (only for one user).

### ISSUE THREAD

**Issue by nashe** [ungdev/EtuUTT#15]
One user reported an 500 on his profile page - reproductible only for his account. I'll try to debug it this week :-)

```
[13-Apr-2015 14:59:45] WARNING: [pool www] child 28562 said into stderr: "NOTICE: PHP message: PHP Fatal error:  Uncaught exception 'RuntimeException' with message 'Failed to start the session because headers have already been sent by "/var/www/app/bootstrap.php.cache" at line 1334.' in /var/www/app/cache/prod/classes.php:103"
[13-Apr-2015 14:59:45] WARNING: [pool www] child 28562 said into stderr: "Stack trace:"
[13-Apr-2015 14:59:45] WARNING: [pool www] child 28562 said into stderr: "#0 /var/www/app/cache/prod/classes.php(169): Symfony\Component\HttpFoundation\Session\Storage\NativeSessionStorage->start()"
[13-Apr-2015 14:59:45] WARNING: [pool www] child 28562 said into stderr: "#1 /var/www/app/cache/prod/classes.php(473): Symfony\Component\HttpFoundation\Session\Storage\NativeSessionStorage->getBag('attributes')"
[13-Apr-2015 14:59:45] WARNING: [pool www] child 28562 said into stderr: "#2 /var/www/src/Etu/Core/CoreBundle/Framework/Listener/TerminateListener.php(50): Symfony\Component\HttpFoundation\Session\Session->set('etu.last_url', '/uvs/category/e...')"
[13-Apr-2015 14:59:45] WARNING: [pool www] child 28562 said into stderr: "#3 [internal function]: Etu\Core\CoreBundle\Framework\Listener\TerminateListener->onKernelTerminate(Object(Symfony\Component\HttpKernel\Event\PostResponseEvent))"
[13-Apr-2015 14:59:45] WARNING: [pool www] child 28562 said into stderr: "#4 /var/www/app/cache/prod/classes.php(1670): call_user_func(Array, Object(Symfony\Component\HttpKernel\Event\PostResponseEvent))"
[13-Apr-2015 14:59:45] WARNING: [pool www] child 28562 said into stderr: "#5 /var/www/app/cache/prod..."
```

**Comment by PunKeel** [comment]
Upgrading Symfony might help (Stable version: 2.8, used version: 2.3)

**Comment by nashe** [comment]
The LTS branch is still symfony 2.3 and we can't / don't want to upgrade before
fixing all the issues (they are not all referenced here..). If it's a bug wich
is fixed in 2.8, I think it would be backported in 2.3?

Anyway, some tests with PHP5.6 are failing and it may be fixed by updating to
the last version of the LTS release. I'll try it in few days!

On ven., sept. 25, 2015 at 11:42 AM, PunKeel < notifications@github.com [notifications@github.com] > wrote:
Upgrading Symfony might help (Stable version: 2.8, used version: 2.3)

—
Reply to this email directly or view it on GitHub
[https://github.com/ungdev/EtuUTT/issues/15#issuecomment-143173413] .[https://github.com/notifications/beacon/AFlW6p31z33agQmstDmo9FFnFgQSHwqpks5o1Q6GgaJpZM4D_ZDW.gif]

**Comment by PunKeel** [comment]
I removed my comment when I remembered 2.3 was the LTS release. If it's a bug in 2.8, it should be backported, but as you don't git composer.lock there is no way to know if deps are up to date
